BI_Tools/Functions/fastq_filter_funcs.py

```diff
@@ -0,0 +1,40 @@
+def get_gc_score(seq: str) -> float:
+    '''
+    Function gc_score, counts GC bound of sequence
+
+    Args: str
+
+    Returns: float
+    '''
+    gc_score = (seq.lower().count("g") + seq.lower().count("c"))*100/len(seq)
+    return gc_score
+
```

Comment:
хороший, рабой код)
стоит добавить проверка на пустую строку
```suggestion
if len(seq) == 0:
    raise ValueError("Sequence length cannot be zero.")
```